### ISSUE TITLE
cordova android build errors

### ISSUE THREAD

**Issue by achmadk** [OneSignal/OneSignal-Cordova-SDK#81]
I use OneSignal plugin for cordova. When I build cordova app in android, Android Studio display 10 errors (image attached). Please fix the errors.
![onesignal error](https://cloud.githubusercontent.com/assets/8057028/18900124/df983518-8568-11e6-8017-fcff4c0b6a8c.PNG)

**Comment by yingliangzhang** [comment]
Same here.
```
/platforms/android/src/com/plugin/gcm/OneSignalPush.java:47: error: package con.onesignal.OneSignal does not exist
import con.onesignal.OneSignal.OSNotification;
                              ^
/platforms/android/src/com/plugin/gcm/OneSignalPush.java:48: error: package con.onesignal.OneSignal does not exist
import con.onesignal.OneSignal.OSNotificationOpenResult;
                              ^
/platforms/android/src/com/plugin/gcm/OneSignalPush.java:288: error: cannot find symbol
    public void notificationReceived(OSNotification notification) {
                                     ^
  symbol:   class OSNotification
  location: class OneSignalPush.CordovaNotificationReceivedHandler
/platforms/android/src/com/plugin/gcm/OneSignalPush.java:306: error: cannot find symbol
    public void notificationOpened(OSNotificationOpenResult result) {
                                   ^
  symbol:   class OSNotificationOpenResult
  location: class OneSignalPush.CordovaNotificationOpenedHandler
/platforms/android/src/com/plugin/gcm/OneSignalPush.java:117: error: incompatible types: String cannot be converted to int
        int appId = data.getString(0);
                                  ^
/platforms/android/src/com/plugin/gcm/OneSignalPush.java:118: error: incompatible types: String cannot be converted to int
        int googleProjectNumber = data.getString(1);
                                                ^
/platforms/android/src/com/plugin/gcm/OneSignalPush.java:122: error: incompatible types: int cannot be converted to String
        appId,
        ^
/platforms/android/src/com/plugin/gcm/OneSignalPush.java:129: error: incompatible types: JSONObject cannot be converted to int
        int displayOption = data.getJSONObject(3);
                                              ^
/platforms/android/src/com/plugin/gcm/OneSignalPush.java:210: error: cannot find symbol
        OneSignal.enableNotificationsWhenActive(data.getBoolean(0));
                 ^
  symbol:   method enableNotificationsWhenActive(boolean)
  location: class OneSignal
```

**Comment by jkasten2** [comment]
@achmadk @yingliangzhang Sorry about the issues with the 2.0 version.  Please run the following to return to the 1.13.2 release.
```
cordova plugin rm onesignal-cordova-plugin
cordova plugin add onesignal-cordova-plugin
```

`cordova plugin list` should now show 1.13.2.

I'll update this issue thread when the 2.0 release is complete.

**Comment by yingliangzhang** [comment]
@jkasten2 Thanks for the quick reply!

**Comment by jkasten2** [comment]
@achmadk @yingliangzhang We have released our 2.0 SDK and can follow our new setup guide below.
https://documentation.onesignal.com/docs/cordova-sdk-setup

You may need to remove and re-add our plugin to get it to update.

Thanks.
